syft/frameworks/torch/hook/hook.py

```diff
@@ -177,6 +180,12 @@ def __init__(
         syft.local_worker = self.local_worker
         syft.hook = self
 
+    def create_wrapper(cls, child_to_wrap):
+        return torch.Tensor()
+
+    def create_shape(cls, shape_dims):
```

Comment:
why is cls a parameter to this method?

Reply:
overrides a classmethod in the base class